schema/sqitch.plan

```diff
@@ -200,4 +200,6 @@ roles/materialized-view-refresher/grants [roles/materialized-view-refresher/crea
 shipping/age-bin-decade [shipping/schema types/intervalrange] 2020-08-12T22:33:55Z Jover Lee <joverlee@fredhutch.org> # Add table representation of decade age bins
 @2020-08-12 2020-08-12T22:50:50Z Jover Lee <joverlee@fredhutch.org> # Schema as of 12 August 2020
 
+functions/barcode_slices 2020-11-05T17:50:20Z Chris Craft <jccraft@uw.edu> # Add function to create slices out of input text
+warehouse/identifier/slices 2020-11-05T20:05:29Z Chris Craft <jccraft@uw.edu> # Add slices column to warehouse.identifier table
 warehouse/identifier/triggers/barcode-distance-check [warehouse/identifier/triggers/barcode-distance-check@2020-08-12] 2020-11-02T20:56:03Z Chris Craft <jccraft@uw.edu> # Making barcode distance check optimizations
```

Comment:
FYI, in the past we've tried to match the order between git commits and the sqitch.plan. I can see it being very confusing in the future to see these out of order in the git history.

Reply:
Good point. I combined these into one commit instead of updating the same .sql file across two commits.